frontend/src/hooks/useExceptionHandler.jsx

```diff
@@ -14,10 +14,11 @@ const useExceptionHandler = () => {
         type: "error",
         content: errMessage,
         title: title,
+        duration: 0,
```

Comment:
@jagadeeswaran-zipstack, I've noticed that we're currently hardcoding the `duration` value to `0` for error notifications, which means that regardless of any passed `duration` value, errors always use `0`. Although our current requirement specifies a `duration` of `0` for all error notifications, it would be beneficial to make this customizable for potential future changes.

Here's how we can handle the error notification conditions:

1. The default duration value for error notifications should be 0.
2. If a duration value is passed to the handleException function, then use that specific value for the duration.